File: eTierMeeting_v2/API/Models/VW_Production_T1_STF_Delivery_Record.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace eTierV2_API.Models
{
    [Keyless]
    public partial class VW_Production_T1_STF_Delivery_Record
    {
        [Column(TypeName = "datetime")]
        public DateTime? Plan_Start_ASY { get; set; }
        [StringLength(3)]
        public string Line_ID_ASY { get; set; }
        [StringLength(3)]
        public string Line_ID_STF { get; set; }
        [StringLength(15)]
        public string MO_No { get; set; }
        [StringLength(3)]
        public string MO_Seq { get; set; }
        [StringLength(40)]
        public string Style_Name { get; set; }
        [StringLength(10)]
        public string Color_No { get; set; }
        public int? Plan_Qty { get; set; }
        public int? Output_Qty { get; set; }
        public int? Output_Balance { get; set; }
        public int? Transfer_Qty { get; set; }
        public int? Transfer_Balance { get; set; }
        [StringLength(10)]
        public string Building { get; set; }
    }
}

File: eTierMeeting_v2/SPA/src/app/_core/_models/production/T1/STF/vW_Production_T1_STF_Delivery_Record.ts
export interface VW_Production_T1_STF_Delivery_Recor {
    plan_Start_ASY: string | null;
    line_ID_ASY: string;
    line_ID_STF: string;
    mO_No: string;
    mO_Seq: string;
    style_Name: string;
    color_No: string;
    plan_Qty: number | null;
    output_Qty: number | null;
    output_Balance: number | null;
    transfer_Qty: number | null;
    transfer_Balance: number | null;
    building: string;
}